ggu/main.c

```diff
@@ -591,60 +591,74 @@ submit_ggu(struct rte_mbuf **pkts, unsigned int num_pkts,
 	return ret;
 }
 
+static inline void
+process_back_nic(struct ggu_config *ggu_conf,
```

Comment:
This function is not really that small to make it `inline`, so drop the tag.

I understand that it's used only once, so it's okay to make it `inline`. But given that it's also `static`, the compiler can also see that and make it `inline` automatically. If this function is used a second time, the compiler will do a better job if we don't include the tag `inline`.